src/Illuminate/Database/Query/Processors/MySqlProcessor.php

```diff
@@ -5,15 +5,26 @@
 class MySqlProcessor extends Processor
 {
     /**
-     * Process the results of a column listing query.
+     * Process the results of columns query.
      *
      * @param  array  $results
      * @return array
      */
-    public function processColumnListing($results)
+    public function processColumns($results)
```

Comment:
Changing the behavior of the `processColumnListing` method on each of these grammars is a breaking change unfortunately. Can we move this to master for Laravel v11?

Reply:
We can keep them without change if you think it's breaking. I'll push a commit.